sdk/appcenter/src/main/java/com/microsoft/appcenter/ingestion/models/AbstractLog.java

```diff
@@ -188,31 +188,27 @@ public void read(JSONObject object) throws JSONException {
         }
     }
 
-    @Override
     @SuppressWarnings("SimplifiableIfStatement")
+    @Override
     public boolean equals(Object o) {
-        if (this == o) {
-            return true;
-        }
-        if (o == null || getClass() != o.getClass()) {
-            return false;
-        }
+        if (this == o) return true;
```

Comment:
Are there any exceptions to "Use `{}` even if you have single operation in block."?

Reply:
If you generate equals/hashcode yes. Its' owned by idea code generator so we don't want to deal with that.

Reply:
Since it's generated code this is also why we ignore "simplify if statement" with an annotation.